vos/vos/vos.py

```diff
@@ -1404,8 +1404,8 @@ def __init__(self, vospace_certfile=None, root_node=None, conn=None,
         :type transfer_shortcut: bool
         :param http_debug: turn on http debugging.
         :type http_debug: bool
-        :param secure_get: Use HTTPS: ie. transfer contents of files using
-        SSL encryption.
+        :param secure_get: Use HTTPS (by default): ie. transfer contents of
+        files using SSL encryption. For future expansion, currently not used.
```

Comment:
Future expansion? I would make it True by default but try http when False (in case it is used with other vospace services that support http).
The description should say that this is used for the more and more unlikely case when the service supports unsecure (HTTP) transfer. And the code should send just http protocols.

Reply:
It is set to True by default (see __init__(..., secure_get = True, ...). 
Updated comments. Currently HTTP is used when secure_get = False.